gap/cryst.gi

```diff
@@ -330,12 +339,19 @@ function ( S, conj )
     if HasWyckoffPositions( S ) then
         W := [];
         for w in WyckoffPositions( S ) do
+          if w!.basis = [] then
+            r := rec( basis       := w!.basis,
```

Comment:
One more completely optional suggestion: instead of duplicating the definition of `r`, you could just change `basis       := w!.basis*c,` to `basis       := w!.basis,` in the definition, and then afterwards insert:
```gap
  if r.basis <> [] then r.basis := r.basis * c; fi;`
```
Anyway, feel free to ignore (and it's not up to me to decide what's merged here anyway)

Reply:
That is far more elegant, thank you.